python_src/src/lamp_py/migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from logging.config import fileConfig
+
+from alembic import context
+
+from lamp_py.postgres.postgres_utils import get_local_engine
+from lamp_py.runtime_utils.import_env import load_environment
+
+# load local environment, if running alembic locally
+load_environment()
```

Comment:
Running alembic locally requires the correct environmental variables to be loaded. There might be a better way to handle this?